.travis/apisix_cli_test.sh

```diff
@@ -458,6 +458,64 @@ fi
 
 echo "passed: found 'my_dict' in nginx.conf"
 
+# allow injecting configuration snippets
+
+echo '
+apisix:
+    node_listen: 9080
+    enable_admin: true
+    port_admin: 9180
+    stream_proxy:
+        tcp:
+            - 9100
+nginx_config:
```

Comment:
How about this style? I think one level is enough and it looks simpler.

```yaml
nginx_config:
    main_configuration_snippet: |
        daemon on;
    http_configuration_snippet: |
        chunked_transfer_encoding on;
    http_server_configuration_snippet: |
        set $my "var";
    http_admin_configuration_snippet: |
        log_format admin "$request_time $pipe";
    stream_configuration_snippet: |
        tcp_nodelay off;

Reply:
@membphis 
Changed.